stream-chat-android-client/src/main/java/io/getstream/chat/android/client/plugins/requests/ApiRequestsDumper.kt

```diff
@@ -0,0 +1,74 @@
+package io.getstream.chat.android.client.plugins.requests
+
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+private const val NOT_FOUND = "not found"
+
+internal class ApiRequestsDumper(
+    private val dateFormat: DateFormat = SimpleDateFormat("HH:mm:ss.SSS", Locale.getDefault())
+) : ApiRequestsAnalyser {
+
+    private val requestsDataMap: MutableMap<String, MutableList<RequestData>> = mutableMapOf()
```

Comment:
What about adding a `clearDataCacheByName` or `clearAllDataCache` interface/method for the `ApiRequestsAnalyser`? The message cache can be huge and that makes it difficult to track specific requests.